Update Dashboard extensions documentation with OCI installation via Rancher UI (#375)
suse-edge/suse-edge.github.io: asciidoc/components/rancher-dashboard-extensions.adoc

== Installation

All SUSE Edge 3.1 components including dashboard extensions are distributed as OCI artifacts. To install SUSE Edge Extensions you can use Rancher Dashboard UI, Helm or Fleet:

=== Installing with Rancher Dashboard UI

> 1. Click *Extensions* in the *Configuration* section of the navigation sidebar.

Reviewer: For numbering, instead of manual numbering, replace "1." with just "." "2." with ".", etc,. It will automatically provide the number. 
For example: 

````
. Click *Extensions* in the *Configuration* section of the navigation sidebar.
````

Author: Done

> 2. On the Extensions page click the three dot menu at the top right and select *Manage Repositories*

Reviewer: ```suggestion
2. On the Extensions page, click the three dot menu at the top right and select *Manage Repositories*.
```

Author: Done

+

> Each extension is distributed via it's own OCI artefact. Therefore it is required to add repositories for each extension we want to install.

Reviewer: For consistency, can we either use either artefact or artifact? 

Also, jut another suggestion: 

````
Each extension is distributed via it's own OCI artefact. Therefore, you need to add repositories for each extension that needs to be installed. 
````

Author: Done

> 3. On the *Repositories page* click `Create` button.

Reviewer: ```suggestion
3. On the *Repositories page*, click `Create`.
```

Author: Done

> 4. In the form, specify the repository name and OCI artefact URL and click `Create`

Reviewer: ```suggestion
4. In the form, specify the repository name and OCI artefact URL, and click `Create`
```

Author: Done

> 4. In the form, specify the repository name and OCI artefact URL and click `Create`

Reviewer: ```suggestion
4. In the form, specify the repository name and OCI artefact URL and click `Create`.
```

Author: Done

+
Akri Dashboard Extension Repository URL:
`oci://registry.suse.com/edge/3.1/akri-dashboard-extension-chart`
+
KubeVirt Dashboard Extension Repository URL:
`oci://registry.suse.com/edge/3.1/kubevirt-dashboard-extension-chart`
+
image::dashboard-extensions-create-oci-repository.png[]

> 5. The extension repository is added to the list and showing the `Active` state.

Reviewer: Is there any action for the user here? Apart from seeing the status? 

```suggestion
. You can see that the extension repository is added to the list and is in `Active` state.
```

Author: No subsequent action here.

+
image::dashboard-extensions-repositories-list.png[]

6. Navigate back to the *Extensions* in the *Configuration* section of the navigation sidebar.
+
In the *Available* tab you can see the extensions available for installation.
+
image::dashboard-extensions-available-extensions.png[]

7. On the extension card click `Install` and confirm the installation.
+

> Once the extension is installed Rancher UI will prompt to reload the page as described in the `https://ranchermanager.docs.rancher.com/integrations-in-rancher/rancher-extensions#installing-extensions[Installing Extensions Rancher documentation page]`.

Reviewer: ```suggestion
Once the extension is installed Rancher UI prompts to reload the page as described in the `https://ranchermanager.docs.rancher.com/integrations-in-rancher/rancher-extensions#installing-extensions[Installing Extensions Rancher documentation page]`.
```
